spec/models/input_element_spec.rb

```diff
@@ -77,4 +77,13 @@
       end
     end
   end
+
+  describe '#sanitized_description' do
+    subject do
+      ie = described_class.new(description: { 'content_en' => 'foobar' })
+      ie.sanitized_description
+    end
+
+    it { is_expected.to eq 'foobar' }
```

Comment:
This isn't really testing the behaviour of `sanitized_description` as there's nothing in the input string "foobar" which would be sanitized.